doc/source/whatsnew/v1.2.0.rst

```diff
@@ -281,7 +281,8 @@ ExtensionArray
 
 Other
 ^^^^^
--
+
+- Removed ``private_key`` and ``verbose`` from :func:`pandas.read_gbq` (:issue:`34654` :issue: `30200`)
```

Comment:
```suggestion
- Removed ``private_key`` and ``verbose`` from :func:`read_gbq` (:issue:`34654` :issue:`30200`)
```

Reply:
Great - thanks for the callout

Reply:
sorry, seems there is still a space between `:issue:` and `30200` now? 🤔